### PR TITLE
ml/python: fix digits samples

### PR DESCRIPTION
**Pull request by berak**
fix loading serialized SVM / KNearest and findContours() usage

### CODE REVIEW

File: samples/python/digits_video.py [hunk 1]
```diff
@@ -41,7 +45,7 @@ def main():
 
         bin = cv.adaptiveThreshold(gray, 255, cv.ADAPTIVE_THRESH_MEAN_C, cv.THRESH_BINARY_INV, 31, 10)
         bin = cv.medianBlur(bin, 3)
-        _, contours, heirs = cv.findContours( bin.copy(), cv.RETR_CCOMP, cv.CHAIN_APPROX_SIMPLE)
+        contours, heirs = cv.findContours( bin.copy(), cv.RETR_CCOMP, cv.CHAIN_APPROX_SIMPLE)
```
Comment (alalek) [inline samples/python/digits_video.py]: OpenCV 3.4.x still uses 3 outputs (due compatibility reasons).

Please apply this fix on master branch (if applicable).

Comment (berak) [reply samples/python/digits_video.py]: ah, right, will do.

Comment (alalek) [reply samples/python/digits_video.py]: I mean separate patch. Keep this for 3.4.

BTW, patch for master is not needed ([code](https://github.com/opencv/opencv/blob/4.2.0/samples/python/digits_video.py#L44)).

**Comment by alalek** [comment]
> +22 -12

Both patches are similar, so patch onto 3.4 is enough (we will merge these changes into "master" by regular "merge 3.4" process).

Closing this.

**Comment by berak** [comment]
yea, right. thanks !

**Pull request closed without merging**